screens/quiz-categories/category.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="./category.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
```

Comment:
You can add these stylesheets in a common file and import that one in all html files

Reply:
Thank you for suggestion, I will do it later while clubbing all files together